content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+the event. The booths present in the :guilabel:`Unavailable` stage have already been purchased, and
+are no longer available.
+
+To modify any existing booth, simply click the desired booth from the :guilabel:`Booths` page, and
+proceed to make any necessary changes from the booth form. Or, create a new one, by clicking the
+:guilabel:`New` button in the upper-left corner to reveal a blank booth form.
+
+Booth form
+----------
+
+The booth form in Odoo *Events* lets users customize and configure event booths in a number of
+different ways.
+
+.. image:: event_booths/booth-form.png
+   :align: center
+   :alt: Typical booth form in the Odoo Events application.
+
+Start by typing in a :guilabel:`Name` for the booth. This is a **required** field.
+
+Then, apply a :guilabel:`Booth Category` to the booth. This is a **required** field.
+
+.. tip::
+   A new :guilabel:`Booth Category` can  be created from this field, by typing in the name of the
+   new category, and clicking :guilabel:`Create and edit...` from the resulting drop-down menu.
+   Doing so reveals a :guilabel:`Create Booth Category` pop-up window, with all the standard fields
+   found on a common booth category form.
+
+   Simply clicking :guilabel:`Create` from the resulting drop-down menu creates the category, but
+   does not reveal the :guilabel:`Create Booth Category` pop-up window. The category would have to
+   be modified later, via the *Booth Categories* page (:menuselection:`Events app --> Configuration
+   --> Booth Categories`).
+
+Upon selecting a pre-existing :guilabel:`Booth Category`, two additional, non-modifiable fields
+appear: :guilabel:`Product` and :guilabel:`Price`. Both fields represent their respective selections
+for that specific booth category.
+
+When a person purchases a booth rental through the event website, the subsequent renter-related
+fields on the form auto-populate, based on the information provided by the purchaser during the
+online transaction. The booth also automatically changes its status from *Available* to
+*Unavailable*.
+
+However, if the rental of a booth is conducted in any other way (e.g., in person, via sales order,
+etc.), the :guilabel:`Renter`, :guilabel:`Renter Name`, :guilabel:`Renter Email`, and
+:guilabel:`Renter Phone` fields can be entered in manually.
+
+The status of the booth can also be changed manually, either by clicking the appropriate status
+from the status bar present on the booth form, or by dragging-and-dropping the desired booth into
+the appropriate stage, via the *Booths* page Kanban view.
+
+Sell event booths
+=================
+
+With event booths configured in the back-end, via the event-specific *Booths* pages, Odoo presents
+them on the front-end of the event website, via the *Get A Booth* event subheader link.
+
+To access the *Get A Booth* page on the event website, open the :menuselection:`Events app`, and
+select the desired event from the :guilabel:`Events` dashboard. From the event form, click the
```

Comment:
If the "Go to Website" smart button is not available, and users see the "Unpublished" smart button, what would they do? :eyes:

Reply:
In my experience, the 'Go to Website' smart button is routinely available, however, as you've mentioned, it can oftentimes be Red/Unpublished. Fortunately, that doesn't affect the ability to click that smart button to reveal the front-end of the event website. So, taking that into consideration, I think I'll be able to keep this as-is, mainly because I believe the concept of 'publishing' an event for all to see is covered in another doc. But, once again, I sincerely appreciate the comment and your amazing attention to detail. Thanks! 👍